### PR TITLE
[SYTO-1097] Mobile Galaxy S6, Video Modal  is not showing correcty

### PR DESCRIPTION
**Pull request by sebasrgl**
https://jira.natgeo.com/browse/SYTO-1097

####How to test
1. ``gulp serve``
2. Scroll down until the Video Promo card
3. Use a mobile emulator and see if the modal render properly.

### CODE REVIEW

File: src/modules/sharing/docs.html [hunk 1]
```diff
@@ -28,8 +28,9 @@ <h3 class="mt3_h3">Sharing <span class="badge">In Development</span></h3>
   }
   &#x3C;/script&#x3E;
 &#x3C;/div&#x3E;</code></pre>
-		<p class="mt3_subh4">The sharing component does not handle the display of the preview, it simply passes the current URL to the corresponding social network. </p>
-    <table class="mt3_subh4 mt3_row-gut-2">
+  <div class="mt3_col-sm-0">
```
Comment (hkekeli) [inline src/modules/sharing/docs.html]: This classname isn't doing anything

File: src/modules/sharing/docs.html [hunk 2]
```diff
@@ -28,8 +28,9 @@ <h3 class="mt3_h3">Sharing <span class="badge">In Development</span></h3>
...
+    <p class="mt3_subh4">The sharing component does not handle the display of the preview, it simply passes the current URL to the corresponding social network. </p>
+    <table class="mt3_subh4 mt3_row-gut-2 mt3_table fixed">
```
Comment (hkekeli) [inline src/modules/sharing/docs.html]: Should 'table' be part of the code base now? You could style it  'src/site/styles/docs

File: src/site/_styles/docs.scss [hunk 3]
```diff
@@ -560,3 +560,9 @@ $snippet-chevron-up: url('data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fww
   margin-top: -9.375rem;
   display: block;
 }
+
+.table-fixed {
+  table-layout: fixed;
+  width: 100%;
+  word-wrap: break-word;
+}
```
Comment (hkekeli) [inline src/site/_styles/docs.scss]: Please move this declaration under:

```
// ====================
// ====================
// Components page
// ====================
// ====================
```

Comment (sebasrgl) [inline src/site/_styles/docs.scss]: Done!

**Comment by pmourelle** [comment]
Seems to be fixed for tables with this new class. So, all tables should have this class set? Wouldn't be better having this styles applied as a normalizer for `table` using max-width: 100% ?

**Comment by sebasrgl** [comment]
Sure sir!

File: src/site/_styles/docs.scss [hunk 4]
```diff
@@ -16,6 +16,9 @@ body {
 
 table {
   border-collapse: collapse;
+  table-layout: fixed;
+  width: 100%;
+  word-wrap: break-word;
 }
```
Comment (hkekeli) [inline src/site/_styles/docs.scss]: Perfect.

**Pull request merged**